django_scrubber/__init__.py

```diff
@@ -29,6 +29,9 @@
         'sites.Site',
         'django_scrubber.FakeData',
     ],
+    'SCRUBBER_VALIDATION_WHITELIST': [
+        'db.TestModel',  # Test model from Django core
```

Comment:
Where is this test model coming from? Never seen that. Can you please provide a link?

Reply:
I didn't know that as well but when you ask Django for "all models", this shows up. Since I don't want people to have to write a scrubber mapping for this one, I'm leaving this out by default.